apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -0,0 +1,167 @@
+                          </a>
+                        </td>
+                        <td>${question.qid}</td>
+                        <td class="text-center">
+                          ${question.max_auto_points
+                            ? resLocals.assessment.type === 'Exam'
+                              ? (question.points_list || [question.max_manual_points ?? 0])
+                                  .map((p) => p - (question.max_manual_points ?? 0))
+                                  .join(',')
+                              : (question.init_points ?? 0) - (question.max_manual_points ?? 0)
+                            : '—'}
+                        </td>
+                        <td class="text-center">${question.max_manual_points || '—'}</td>
+                        <td class="text-center" data-testid="iq-to-grade-count">
+                          ${question.num_instance_questions_to_grade} /
+                          ${question.num_instance_questions}
+                        </td>
+                        <td>
+                          ${progressBar(
+                            question.num_instance_questions_to_grade,
+                            question.num_instance_questions,
+                          )}
+                        </td>
+                        <td>
+                          ${question.num_instance_questions_assigned > 0
+                            ? html`<strong class="bg-warning rounded px-1">
+                                  ${resLocals.authz_data.user.name ??
+                                  resLocals.authz_data.user.uid}</strong
+                                >${question.assigned_graders?.some((u) =>
+                                  idsEqual(u.user_id, resLocals.authz_data.user.user_id),
+                                )
+                                  ? ', '
+                                  : ''}`
+                            : ''}
+                          ${(question.assigned_graders || [])
+                            .filter((u) => !idsEqual(u.user_id, resLocals.authz_data.user.user_id))
+                            .map((u) => u.name ?? u.uid)
+                            .join(', ')}
+                          ${question.num_instance_questions_unassigned > 0
+                            ? html` <small class="text-muted"
+                                >(${question.num_instance_questions_unassigned} unassigned)</small
+                              >`
+                            : ''}
+                        </td>
+                        <td>
+                          ${(question.actual_graders || []).map((u) => u.name ?? u.uid).join(', ')}
+                        </td>
+                        <td>
+                          ${resLocals.authz_data.has_course_instance_permission_edit &&
+                          question.num_instance_questions_assigned +
+                            question.num_instance_questions_unassigned >
+                            0
```

Comment:
Sometimes it's nice to move these conditionals out of the middle of the template string and out into the enclosing function (in this case the `.map(...)` callback). E.g.:

```js
const showGradeNextSubmissionButton = resLocals.authz_data.has_course_instance_permission_edit && question.num_instance_questions_assigned + question.num_instance_questions_unassigned > 0;

return html`
  ...
  ${showGradeNextSubmissionButton ? html`...` : ''}
  ...
`
```

Reply:
Done in 183a7e7ef.